ddp-workspace/projects/ddp-rgp/src/styles/_activity.scss

```diff
@@ -229,9 +229,9 @@ ddp-activity-dropdown-picklist-question select {
   border: none;
   border-bottom: 1px solid rgba(0, 0, 0, 0.42);
 
-  &:disabled {
-    border-bottom-style: dotted;
-  }
+  // &:disabled {
+  //   border-bottom-style: dotted;
+  // }
```

Comment:
I noticed an extra dotted line when in read-only mode, so removing this.

![rgp_bug_dotted](https://user-images.githubusercontent.com/5713833/116941929-9bfe0d80-ac3e-11eb-8891-b78637765954.png)